build_scripts/windows/scripts/az.ps1

```diff
@@ -1,217 +1,217 @@
 $env:AZ_INSTALLER="MSI"
-& "$PSScriptRoot\..\python.exe" -IBm azure.cli $args
+& "$PSScriptRoot\..\python.exe" -IBm azure.cli @args
```

Comment:
The disadvantage of using `@args` is also obvious, as we can't pass `"a b"` string as `("a", "b")` now.